math/ensmallen/Portfile

```diff
@@ -23,12 +22,14 @@ long_description \
 homepage            https://ensmallen.org
 master_sites        ${homepage}/files
 
-checksums           rmd160  546c4683d3eaf21d84866daade8f88724c90dcde \
-                    sha256  e496c207c20816150a535f891172357f3c136ad8e73cbbc01a77112eb38b0c9a \
-                    size    1265213
+checksums           rmd160  e469e8a4b8fc8038f30bbfa4e49351e6ee3a1269 \
+                    sha256  f9b280e03266b308d2772ab7e1dd962adcfa98b0552b05795f8ac81ec15eb270 \
+                    size    1328515
 
 depends_lib         port:armadillo
 
+compiler.cxx_standard 2011
+
 cmake.out_of_source yes
```

Comment:
this line is the default in the `cmake` 1.1 PG so should be removed since you updated that above

Reply:
> this line is the default in the `cmake` 1.1 PG so should be removed since you updated that above

@reneeotten Let me check is that works as expected. From what I recall (not about this port specifically) Cmake, having that PG version set, may choose gcc-4.2 or Xcode clang on 10.6, neither of which support C++11.

Reply:
@reneeotten So no, it does not work. Without `compiler.cxx_standard 2011` Cmake picks `gcc-4.2`, which obviously fails.

```
--->  Configuring ensmallen
Executing:  cd "/opt/local/var/macports/build/_opt_PPCRosettaPorts_math_ensmallen/ensmallen/work/build" && /opt/local/bin/cmake -G "CodeBlocks - Unix Makefiles" -DCMAKE_BUILD_TYPE=MacPorts -DCMAKE_INSTALL_PREFIX="/opt/local" -DCMAKE_INSTALL_NAME_DIR="/opt/local/lib" -DCMAKE_SYSTEM_PREFIX_PATH="/opt/local;/usr" -DCMAKE_C_COMPILER="$CC" -DCMAKE_CXX_COMPILER="$CXX" -DCMAKE_OBJC_COMPILER="$CC" -DCMAKE_OBJCXX_COMPILER="$CXX" -DCMAKE_POLICY_DEFAULT_CMP0025=NEW -DCMAKE_POLICY_DEFAULT_CMP0060=NEW -DCMAKE_VERBOSE_MAKEFILE=ON -DCMAKE_COLOR_MAKEFILE=ON -DCMAKE_FIND_FRAMEWORK=LAST -DCMAKE_EXPORT_COMPILE_COMMANDS=ON -DCMAKE_MAKE_PROGRAM=/usr/bin/make -DCMAKE_MODULE_PATH="/opt/local/share/cmake/Modules" -DCMAKE_PREFIX_PATH="/opt/local/share/cmake/Modules" -DCMAKE_BUILD_WITH_INSTALL_RPATH:BOOL=ON -DCMAKE_INSTALL_RPATH="/opt/local/lib" -Wno-dev -DARMADILLO_INCLUDE_DIR=/opt/local/include -DARMADILLO_LIBRARY=/opt/local/lib/libarmadillo.dylib -DCMAKE_OSX_ARCHITECTURES="ppc" -DCMAKE_OSX_DEPLOYMENT_TARGET="10.6" -DCMAKE_OSX_SYSROOT="/" /opt/local/var/macports/build/_opt_PPCRosettaPorts_math_ensmallen/ensmallen/work/ensmallen-2.19.0 
-- The C compiler identification is GNU 4.2.1
-- The CXX compiler identification is GNU 4.2.1
```

Reply:
that is not the line he is talking about

Reply:
@kencu @reneeotten I am sorry for misunderstanding. Will correct that.